sdk/core-rest/core-client/src/pathClientTypes.ts

```diff
@@ -0,0 +1,38 @@
+import { RawHttpHeaders } from "@azure/core-rest-pipeline";
+
+/**
+ * Shape of the default request parameters, this may be overriden by the specific
+ * request types to provide strong types
+ */
+export type RequestParameters = {
+  /**
+   * Headers to send along with the request
+   */
+  headers?: RawHttpHeaders;
+  /**
+   * Body to send with the request
+   */
+  body?: unknown;
+  /**
+   * Query parameters to send with the request
+   */
+  queryParameters?: { [key: string]: any };
```

Comment:
if the body is `unknown`, why the query parameter values are not? I prefer to use `unknown` instead throughout and it would not affect the user experience as long as they are for input stuff but will help us think about the types of things in our implementation.